src/dpsnotificationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsnotificationservice/OutputNotificationEndpoint.java

```diff
@@ -53,7 +51,7 @@ public OutputNotificationResponse outputNotificationNotification(@RequestPayload
             return response;
         }
 
-        if(StringUtils.isBlank(request.getOutputNotificationRequest().getBusinessAreaCd())) {
+        if(request.getOutputNotificationRequest().getBusinessAreaCd() == null) {
```

Comment:
out of PR scope, but are we missing a stack trace in this if?

Reply:
nope, just a normal check no exception here

Reply:
Lets

Reply:
Go